### ISSUE TITLE
IPython 4.0, aka The Big Split

### ISSUE THREAD

**Issue by rosswhitfield** [mantidproject/mantid#13481]
python imports will need to be updated for the new iPython because qtconsole has been separated into jupyter.

You get the following warning
```bash
/usr/lib/python2.7/site-packages/IPython/qt.py:13: ShimWarning: The `IPython.qt` package has been deprecated. You should import from qtconsole instead.
  "You should import from qtconsole instead.", ShimWarning)
/usr/lib/python2.7/site-packages/qtconsole/rich_ipython_widget.py:2: DeprecationWarning: qtconsole.rich_ipython_widget is deprecated. use qtconsole.rich_jupyter_widget
  warnings.warn("qtconsole.rich_ipython_widget is deprecated. use qtconsole.rich_jupyter_widget", DeprecationWarning)
/usr/lib/python2.7/site-packages/qtconsole/rich_jupyter_widget.py:366: UserWarning: RichIPythonWidget is deprecated, use RichJupyterWidget
  warn("RichIPythonWidget is deprecated, use RichJupyterWidget")
```

**Comment by rosswhitfield** [comment]
With the new iPython qtconsole in the _Script Interpreter_ you will now need to `Shift+Enter` instead of just `Enter` to execute a command. This get tiring quickly.

**Comment by rosswhitfield** [comment]
Package dependencies will also need to be updated as `qtconsole` is no longer included in the `ipython` package.

**Comment by rosswhitfield** [comment]
Fix in #17152
